setup.cfg

```diff
@@ -19,6 +19,7 @@ classifiers =
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
```

Comment:
supporting a new python version affects lots of things, definitely do it in a separate PR and enable tests. we will not do it for now.